docs/radosgw-admin.md

```diff
@@ -0,0 +1,32 @@
+# radosgw-admin
+
+This used to run any radosgw-admin cli command with with arbitrary args.
+
+## Examples
+
+```bash
+kubectl rook-ceph radosgw-admin user create --display-name="my user" --uid=myuser
+
+# Info: running 'radosgw-admin' command with args: [user create --display-name=johnny rotten --uid=johnny]
```

Comment:
The output still has jonny rotten :)

Reply:
I just changed the name in the command missed updating the output